Keas.Mvc/Services/HistoryService.cs

```diff
@@ -657,6 +658,22 @@ public async Task<History> DocumentStatusChange(Document document)
             _context.Histories.Add(historyEntry);
             return historyEntry;
         }
+
+        public async Task<History> DocumentDeleted(Document document)
+        {
+            var person = await _securityService.GetPerson(document.TeamId);
```

Comment:
Shouldn't document deletion history store the id of the person performing the deletion and not the generic Team person?

Oh, wait, I might be misunderstanding how securityService.GetPerson works.

Yeah, it looks the choice of real vs generic team user depends on authentication method. So deletions by token-authenticated calls will only show a team user.